app2/src/main/java/com/example/recipebox/Contact.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.recipebox
+
+//java class database
+//users information
+class Contact {
+    var id = 0
+    var name: String? = null
+    var phone_number: String? = null
+
+    constructor() {}
+    constructor(id: Int, name: String?, phone_number: String?) {
+        this.id = id
+        this.name = name
+        this.phone_number = phone_number
+    }
```

Comment:
I can't recommend enough the use of Kotlin "POJO's". It removes a ton of the boilerplate code (setter/getters) from a normal java POJO.

Reply:
Agreed.